vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -42,7 +42,7 @@
       state: shutdown
     register: shutdown_vms
     retries: 50
```

Comment:
As we have decreased the delay I think we need to increase the retries to maintain the same time needed to wait ? 
```suggestion
    retries: 166
```

Reply:
otherwise that many retries may not be enough too :)

Reply:
Thanks, fixed!